modules/videobyteBidAdapter.js

```diff
@@ -0,0 +1,311 @@
+  }
+
+  // Obtain all ORTB params related video from Ad Unit
+  VIDEO_ORTB_PARAMS.forEach((param) => {
+    if (videoParams.hasOwnProperty(param)) {
+      video[param] = videoParams[param];
+    }
+  });
+
+  // Placement Inference Rules:
+  // - If no placement is defined then default to 1 (In Stream)
+  video.placement = video.placement || 2;
+
+  // - If product is instream (for instream context) then override placement to 1
+  if (params.context === 'instream') {
+    video.startdelay = video.startdelay || 0;
+    video.placement = 1;
+  }
+
+  // bid floor
+  const bidFloorRequest = {
+    currency: bidRequest.params.cur || 'USD',
+    mediaType: 'video',
+    size: '*'
+  };
+  let floorData = bidRequest.params
+  if (utils.isFn(bidRequest.getFloor)) {
+    floorData = bidRequest.getFloor(bidFloorRequest);
+  } else {
+    if (params.bidfloor) {
+      floorData = {floor: params.bidfloor, currency: params.currency || 'USD'};
+    }
+  }
+
+  const openrtbRequest = {
+    id: bidRequest.bidId,
+    imp: [
+      {
+        id: '1',
+        video: video,
+        secure: isSecure() ? 1 : 0,
+        bidfloor: floorData.floor,
+        bidfloorcur: floorData.currency
+      }
+    ],
+    site: {
+      domain: window.location.hostname,
+      page: window.location.href,
+      ref: bidRequest.refererInfo ? bidRequest.refererInfo.referer || null : null
+    },
+    ext: {
+      hb: 1,
+      prebidver: '$prebid.version$',
+      adapterver: spec.VERSION,
+    },
+  };
+
+  // content
+  if (videoParams.content && utils.isPlainObject(videoParams.content)) {
+    openrtbRequest.site.content = {};
+    const contentStringKeys = ['id', 'title', 'series', 'season', 'genre', 'contentrating', 'language'];
+    const contentNumberkeys = ['episode', 'prodq', 'context', 'livestream', 'len'];
+    const contentArrayKeys = ['cat'];
+    const contentObjectKeys = ['ext'];
+    for (const contentKey in videoBidderParams.content) {
+      if (
+        (contentStringKeys.indexOf(contentKey) > -1 && utils.isStr(videoParams.content[contentKey])) ||
+        (contentNumberkeys.indexOf(contentKey) > -1 && utils.isNumber(videoParams.content[contentKey])) ||
+        (contentObjectKeys.indexOf(contentKey) > -1 && utils.isPlainObject(videoParams.content[contentKey])) ||
+        (contentArrayKeys.indexOf(contentKey) > -1 && utils.isArray(videoParams.content[contentKey]) &&
+          videoParams.content[contentKey].every(catStr => utils.isStr(catStr)))) {
+        openrtbRequest.site.content[contentKey] = videoParams.content[contentKey];
+      } else {
+        utils.logMessage('videobyte bid adapter validation error: ', contentKey, ' is either not supported is OpenRTB V2.5 or value is undefined');
+      }
+    }
+  }
+
+  // adding schain object
+  if (bidRequest.schain) {
+    utils.deepSetValue(openrtbRequest, 'source.ext.schain', bidRequest.schain);
+    openrtbRequest.source.ext.schain.nodes[0].rid = openrtbRequest.id;
+  }
```

Comment:
You didn't specify in your docs PR that your adapter supports `schain`.

Reply:
documentation is updated 👍